tests/test_entity.py

```diff
@@ -0,0 +1,124 @@
+import json
+import requests
+
+
+def get_entities(auth_header):
+    response = requests.get("http://localhost:5000/entities", headers=auth_header)
+    assert response.status_code == 200
+    return response.json()
+
+
+def get_entity_by_id(auth_header, id):
+    response = requests.get(
+        f"http://localhost:5000/entities/{id}",
+        headers=auth_header,
+    )
+    assert response.status_code == 200
+    return response.json()
+
+
+def get_file(auth_header, filename):
+    response = requests.get(
+        f"http://localhost:5000/entities/files/{filename}",
+        headers=auth_header,
+    )
+    assert response.status_code == 200
+    return response.json()
+
+
+def create_entity(auth_header, body, fs, file):
+    if fs:
+        response = requests.post(
+            f"http://localhost:5000/entities/",
+            headers=auth_header,
+            files={"file": file},
+            data={"body": json.dumps(body)},
+        )
+    else:
+        response = requests.post(
+            f"http://localhost:5000/entities/",
+            json=body,
+            headers=auth_header,
+        )
+    assert response.status_code == 201
+    actual = {k: v for k, v in response.json().items() if k in body}
+    assert actual == body
+    return response.json()
+
+
+def update_entity(auth_header, id, body, fs, file):
+    if fs:
+        response = requests.put(
+            f"http://localhost:5000/entities/{id}",
+            headers=auth_header,
+            files={"file": file},
+            data={"body": json.dumps(body)},
+        )
+    else:
+        response = requests.put(
+            f"http://localhost:5000/entities/{id}",
+            json=body,
+            headers=auth_header,
+        )
+    assert response.status_code == 200
+    actual = {k: v for k, v in response.json().items() if k in body}
+    assert actual == body
+    return response.json()
+
+
+def delete_entity(auth_header, id):
+    response = requests.delete(
+        f"http://localhost:5000/entities/{id}",
+        headers=auth_header,
+    )
+    assert response.status_code == 200
```

Comment:
Note: release branch hasn't been updated with your changes in [#133](https://github.com/uwblueprint/starter-code-v2/pull/133) yet so this will fail when using TypeScript backend (which still returns 204)